app/services/data/get-individual-overview.js

```diff
@@ -15,7 +16,9 @@ module.exports = function (id) {
            'workload_points_calculations.total_points AS totalPoints',
            'workload.total_cases AS cases',
            'workload_owner.contracted_hours AS contractedHours',
-           'workload_points_calculations.reduction_hours AS reduction')
+           'workload_points_calculations.reduction_hours AS reduction',
+           'workload_points.default_contracted_hours_po AS defaultContractedHoursPo',
+           'workload_points.default_contracted_hours_pso AS defaultContractedHoursPso')
```

Comment:
Still not completely sure on best way to proceed with these bands. Currently they only have one default hours for all bands. Do we confine it to one? If we want more, do we need to include all? Just a thought...